### DISCUSSION TITLE
fuel consumtion/costs for route planning

### DISCUSSION THREAD

**Discussion by Cppugino** [osmandapp/OsmAnd#14012] [Maps Related]
Hello dear Omsand developers.

It would be useful to be able to enter and save the average fuel consumption/energy consumption of a vehicle plus the approximate current fuel costs/energy costs in the navigation planning (settings). The goal would be an output of fuel costs at the planned route.

Would this be feasible in the short term?

Thanks a lot! BR Cappugino

**Comment by mariush444** [comment]
https://github.com/osmandapp/OsmAnd/discussions/11943
